geoportailv3/static/js/catalog/catalogdirective.js

```diff
@@ -114,9 +114,7 @@ app.CatalogController.prototype.setTree_ = function() {
        * @param {Object} tree Tree object for the theme.
        */
       function(tree) {
-        if (!goog.isNull(tree)) {
-          this['tree'] = tree;
-        }
```

Comment:
It seems to work.
What will be a better way to notify the layer manager, that the layer tree does not exist anymore ?